pages/programs/baton_roots.html

```diff
@@ -1 +1,121 @@
+<!DOCTYPE html>
+<html lang="en" >
 
+<head>
+  <meta charset="UTF-8">
+      <link rel="stylesheet" href="./youth-managed-farms.css"> <!--CHANGE DIR WHEN CSS SUBMITTED-->
+</head>
+
+<body>
+
+  <body>
+	<div id="headerBR">
+		<div class="grid-st">
+			<img src="http://www.thewallsproject.org/img/shared/walls_logo.svg"  />
+		</div>
+		<div class="grid-end">
+			<div id="navBR">
+				<div class="navBR-1"><a href="#">Progress<span class="u-BR"></span></a></div>
```

Comment:
Can we update this effect to follow the HTML structure that exists on http://ianlunn.github.io/Hover/ and search for the "Underline from center" button